tfx/src/main/scala/com/spotify/noether/tfx/package.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2018 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.noether
+
+import com.spotify.noether.tfx.TfmaConverter
+import com.twitter.algebird.Aggregator
+import tensorflow_model_analysis.MetricsForSliceOuterClass.MetricsForSlice
+
+import scala.language.implicitConversions
+
+package object tfx {
+  type Pred = Prediction[Boolean, Double]
```

Comment:
Might want to rename this to something more specific (e.g. BooleanPred) since there are multiple parameters used for the Prediction type e.g. ConfusionMatrix predictions are Prediction[Int, Int], LogLoss have Prediction[Int, List[Double]]

Reply:
Sure, `BinaryPred` maybe?